<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>500</width>
    <height>510</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>500</width>
    <height>510</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>500</width>
    <height>520</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="Line" name="line_2">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>-1</y>
      <width>2</width>
      <height>501</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Orientation::Vertical</enum>
    </property>
   </widget>
   <widget class="Line" name="line">
    <property name="geometry">
     <rect>
      <x>11</x>
      <y>239</y>
      <width>219</width>
      <height>3</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Orientation::Horizontal</enum>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>11</x>
      <y>248</y>
      <width>48</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>视频列表</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>11</x>
      <y>11</y>
      <width>48</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>节目信息</string>
    </property>
   </widget>
   <widget class="QTableWidget" name="tableWidget_Config">
    <property name="geometry">
     <rect>
      <x>11</x>
      <y>32</y>
      <width>219</width>
      <height>201</height>
     </rect>
    </property>
    <column>
     <property name="text">
      <string>名称</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>栏目ID</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>节目ID</string>
     </property>
    </column>
   </widget>
   <widget class="QTableWidget" name="tableWidget_List">
    <property name="geometry">
     <rect>
      <x>11</x>
      <y>269</y>
      <width>219</width>
      <height>201</height>
     </rect>
    </property>
    <column>
     <property name="text">
      <string>名称</string>
     </property>
    </column>
   </widget>
   <widget class="QPushButton" name="flash_program">
    <property name="geometry">
     <rect>
      <x>205</x>
      <y>6</y>
      <width>25</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset resource="resources.qrc">
      <normaloff>:/resources/flash.png</normaloff>:/resources/flash.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>20</width>
      <height>20</height>
     </size>
    </property>
   </widget>
   <widget class="QPushButton" name="flash_list">
    <property name="geometry">
     <rect>
      <x>205</x>
      <y>242</y>
      <width>25</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset resource="resources.qrc">
      <normaloff>:/resources/flash.png</normaloff>:/resources/flash.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>20</width>
      <height>20</height>
     </size>
    </property>
   </widget>
   <widget class="QLabel" name="label_img">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>10</y>
      <width>240</width>
      <height>141</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::Shape::StyledPanel</enum>
    </property>
    <property name="text">
     <string>暂无图片QAQ</string>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
    <property name="alignment">
     <set>Qt::AlignmentFlag::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_title">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>160</y>
      <width>240</width>
      <height>20</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::Shape::StyledPanel</enum>
    </property>
    <property name="text">
     <string>暂无标题</string>
    </property>
   </widget>
   <widget class="QScrollArea" name="scrollArea">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>190</y>
      <width>240</width>
      <height>170</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::Shape::StyledPanel</enum>
    </property>
    <property name="widgetResizable">
     <bool>true</bool>
    </property>
    <widget class="QWidget" name="scrollAreaWidgetContents">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>238</width>
       <height>168</height>
      </rect>
     </property>
     <widget class="QLabel" name="label_introduce">
      <property name="geometry">
       <rect>
        <x>5</x>
        <y>0</y>
        <width>235</width>
        <height>170</height>
       </rect>
      </property>
      <property name="frameShape">
       <enum>QFrame::Shape::NoFrame</enum>
      </property>
      <property name="text">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;暂无介绍&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>430</y>
      <width>240</width>
      <height>40</height>
     </rect>
    </property>
    <property name="text">
     <string>下载</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_time">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>370</y>
      <width>240</width>
      <height>50</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::Shape::StyledPanel</enum>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;暂无时间&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>500</width>
     <height>20</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>程序</string>
    </property>
    <addaction name="actionfile"/>
    <addaction name="actionsetting"/>
    <addaction name="actionabout"/>
    <addaction name="separator"/>
    <addaction name="actionexit"/>
   </widget>
   <widget class="QMenu" name="menu_2">
    <property name="title">
     <string>节目</string>
    </property>
    <addaction name="actionimport"/>
   </widget>
   <addaction name="menu"/>
   <addaction name="menu_2"/>
  </widget>
  <action name="actionfile">
   <property name="text">
    <string>打开文件保存位置</string>
   </property>
  </action>
  <action name="actionexit">
   <property name="text">
    <string>退出</string>
   </property>
  </action>
  <action name="actionout">
   <property name="text">
    <string>导出节目单</string>
   </property>
  </action>
  <action name="actionsetting">
   <property name="text">
    <string>设置</string>
   </property>
  </action>
  <action name="actionabout">
   <property name="text">
    <string>关于...</string>
   </property>
  </action>
  <action name="actionfrom_file">
   <property name="text">
    <string>从文件导入</string>
   </property>
  </action>
  <action name="actionfrom_link">
   <property name="text">
    <string>从链接导入</string>
   </property>
  </action>
  <action name="actionimport">
   <property name="text">
    <string>导入节目</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="resources.qrc"/>
 </resources>
 <connections/>
</ui>
